evm/src/cpu/kernel/tests/hash.rs

```diff
@@ -38,109 +36,95 @@ fn make_random_input() -> Vec<u8> {
     (0..num_bytes).map(|_| rng.gen()).collect()
 }
 
-fn make_custom_input() -> Vec<u8> {
```

Comment:
Why are we getting rid of this? Is it because it was only really useful for debugging?

Reply:
yeah I did that because William found a bug in my ripeMD impl and this was the bug-causing input